.github/workflows/eas-build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           ssh-private-key: ${{ secrets.GH_SSH_KEY }}
 
       - name: Install dependencies
-        run: yarn install --production
+        run: sudo yarn install --production
```

Comment:
Not sure if this was meant to be changed.

Reply:
Yes, intentional.